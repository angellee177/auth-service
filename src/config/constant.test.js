const Constant = require("./constant");

describe("Constant Test Suite", () => {
    describe("NODE_ENVIRONMENT attributes", () => {
        it("should match snapshot", () => {
            expect(Constant.NODE_ENVIRONMENT).toBeDefined();
            expect(Constant.NODE_ENVIRONMENT).toMatchSnapshot();
        });
    });
});