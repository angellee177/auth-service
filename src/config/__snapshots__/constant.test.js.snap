// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constant Test Suite NODE_ENVIRONMENT attributes should match snapshot 1`] = `
Object {
  "dev": "development",
  "prod": "production",
  "test": "testing",
}
`;
